core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4936,7 +4936,7 @@ private static void computeVersion(ServletContext context) {
         if(ver.equals(UNCOMPUTED_VERSION) || SystemProperties.getBoolean("hudson.script.noCache"))
             RESOURCE_PATH = "";
         else
-            RESOURCE_PATH = "/static/"+SESSION_HASH;
+            RESOURCE_PATH = "static/"+SESSION_HASH;
 
         VIEW_RESOURCE_PATH = "/resources/"+ SESSION_HASH;
```

Comment:
Are you asking about `VIEW_RESOURCE_PATH`? I noticed its leading slash there too but did not do the research to check how it was used. So I couldn't be sure if it was right to change it or not.